### PR TITLE
Amyrman/issue15

### PR DESCRIPTION
**Pull request by amyrman**
Backend lacks an end-point that handles login

### CODE REVIEW

**Comment by amyrman** [comment]
Still got the testing part to do

**Review by stratosfloor — APPROVED** [review]
Test is successfull.
Also tested in Postman with Bearer token and response is the correct userId.

File: backend/src/test/java/com/example/parkingspot/controller/UserControllerMVCTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    @Test
+    void callingEndpointLogin_shouldReturn200OK_andId() throws Exception {
+
+        when(userService.getUserId("dolores@test.nu")).thenReturn("1");
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/login/{email}", "dolores@test.nu"))
+                .andExpect(MockMvcResultMatchers.status().isOk())
+                .andExpect(MockMvcResultMatchers.content().string("1"));
+    }
+}
```
Comment (christerresin) [inline backend/src/test/java/com/example/parkingspot/controller/UserControllerMVCTest.java:38]: Clean test, passes. 

Future test? Should we have a test for invalid email? Check if the endpoint works and behaves as intended

**Review by christerresin — APPROVED** [review]
Pulled and checked locally. Everything works and looks good.

**Pull request merged**